src/kvstore/kvstore_local.h

```diff
@@ -174,13 +175,75 @@ class KVStoreLocal : public KVStore {
 
  protected:
   /**
-   * \brief group values on keys
+   * \brief group values on keys for push
    */
-  template <typename V>
+  void GroupKVPairsPush(const std::vector<int>& keys,
```

Comment:
I also thought about that. What file do you think is the best place to put the free function? I started with `GroupKVPairsForPushRsp` but it is too long (>100 char) causing lint to fail..